EIPS/eip-7704.md

```diff
@@ -0,0 +1,246 @@
+---
+eip: 7704
```

Comment:
```suggestion
eip: 7706
```
Assigning next sequential EIP/ERC/RIP number.  7704 had already been assigned to an ERC.

Please also update the filename